src/octopeer-github/content/ContentController.ts

```diff
@@ -18,6 +18,7 @@ class ContentController {
      * @return this
      */
     public start() {
+        Options.init();
```

Comment:
No, this is the ContentController. It was only initialized in the background page, but now it's also initialized in the contentscript :)